pkg/agent/informant.go

```diff
@@ -236,6 +246,15 @@ func NewInformantServer(
 			case <-ticker.C:
 			}
 
+			// Are we talking to a monitor?
+			if server.informantIsMonitor {
```

Comment:
is it possible for `server.informantIsMonitor` to change after this point? If so, reading the field should be done with a lock acquired.